apps/explorer/src/ui/DescriptionList.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ReactNode } from 'react';
+
+import { Text, type TextProps } from '~/ui/Text';
+
+export function Label(props: TextProps) {
```

Comment:
You should still create and export `LabelProps`, it should just be `export interface LabelProps extends TextProps {}`